frontend/src/pages/projects/screens/detail/storage/ManageStorageModal.tsx

```diff
@@ -84,7 +87,9 @@ const ManageStorageModal: React.FC<AddStorageModalProps> = ({ existingData, isOp
       }
       return;
     }
-    const createdPvc = await createPvc(createData, namespace, { dryRun });
+    const createdPvc = await createPvc(createData, namespace, storageClassName?.metadata.name, {
```

Comment:
We don't need the forth optional parameter, opts, passed as undefined here, right? Cause it it the last optional parameter.